<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyUserOrientation</class>
 <widget class="QWidget" name="PropertyUserOrientation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>843</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User orientation properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>576</width>
        <height>819</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QGroupBox" name="setOrientationGroupBox">
         <property name="title">
          <string>Set Axis (Orientation)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="XRadioButton">
            <property name="text">
             <string>Axis X'</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="YRadioButton">
            <property name="text">
             <string>Axis Y'</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="customAxisRadioButton">
            <property name="text">
             <string>Custom Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="customAxisFrame">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="1">
              <widget class="QDoubleEdit" name="customAxisX2">
               <property name="text">
                <string>1.0</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleEdit" name="customAxisY1">
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Enter coord 2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QDoubleEdit" name="customAxisY2">
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Enter coord 1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleEdit" name="customAxisX1">
               <property name="text">
                <string>0.0</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Pick points along axis</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QDoubleEdit" name="z_p1Edit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QDoubleEdit" name="y_p2Edit">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Z</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleEdit" name="x_p2Edit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleEdit" name="x_p1Edit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="p2Button">
               <property name="text">
                <string>Pick point 2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QDoubleEdit" name="y_p1Edit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QDoubleEdit" name="z_p2Edit">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="p1Button">
               <property name="text">
                <string>Pick point 1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>525</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="setTranslationGroupBox">
         <property name="title">
          <string>Set Coordinates (Translation)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="3">
           <widget class="QDoubleEdit" name="setCoorNewYValue">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QDoubleEdit" name="setCoorNewZValue">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleEdit" name="setCoorOldXValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QDoubleEdit" name="setCoorOldZValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleEdit" name="setCoorOldYValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="setCoorPickPointYButton">
            <property name="text">
             <string>Pick Y Point</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="setCoorPickPointZButton">
            <property name="text">
             <string>Pick Z Point</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="setCoorPickPointXButton">
            <property name="text">
             <string>Pick X Point</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>New Values</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleEdit" name="setCoorNewXValue">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4" rowspan="3">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Current Values</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>XRadioButton</tabstop>
  <tabstop>YRadioButton</tabstop>
  <tabstop>customAxisRadioButton</tabstop>
  <tabstop>customAxisX1</tabstop>
  <tabstop>customAxisY1</tabstop>
  <tabstop>customAxisX2</tabstop>
  <tabstop>customAxisY2</tabstop>
  <tabstop>p1Button</tabstop>
  <tabstop>x_p1Edit</tabstop>
  <tabstop>y_p1Edit</tabstop>
  <tabstop>z_p1Edit</tabstop>
  <tabstop>p2Button</tabstop>
  <tabstop>x_p2Edit</tabstop>
  <tabstop>y_p2Edit</tabstop>
  <tabstop>z_p2Edit</tabstop>
  <tabstop>setCoorPickPointXButton</tabstop>
  <tabstop>setCoorPickPointYButton</tabstop>
  <tabstop>setCoorPickPointZButton</tabstop>
  <tabstop>setCoorOldXValue</tabstop>
  <tabstop>setCoorOldYValue</tabstop>
  <tabstop>setCoorOldZValue</tabstop>
  <tabstop>setCoorNewXValue</tabstop>
  <tabstop>setCoorNewYValue</tabstop>
  <tabstop>setCoorNewZValue</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
